hail/python/hail/vds/methods.py

```diff
@@ -371,9 +371,9 @@ def impute_sex_chromosome_ploidy(
 
      -  ``autosomal_mean_dp`` (*float64*): Mean depth on calling intervals on normalization contig.
      -  ``x_mean_dp`` (*float64*): Mean depth on calling intervals on X chromosome.
-     -  ``x_ploidy`` (*float64*): Estimated ploidy on X chromosome. Equal to ``2 * autosomal_mean_dp / x_mean_dp``.
+     -  ``x_ploidy`` (*float64*): Estimated ploidy on X chromosome. Equal to ``x_mean_dp / autosomal_mean_dp * 2``.
      -  ``y_mean_dp`` (*float64*): Mean depth on calling intervals on  chromosome.
-     -  ``y_ploidy`` (*float64*): Estimated ploidy on Y chromosome. Equal to ``2 * autosomal_mean_dp / y_mean_dp``.
+     -  ``y_ploidy`` (*float64*): Estimated ploidy on Y chromosome. Equal to ``y_mean_db / autosomal_mean_dp * 2``.
```

Comment:
This associates as `(x_mean_dp / autosomal_mean_dp) * 2`, right? Could you write it `2 * x_mean_dp / autosomal_mean_dp` so there's no ambiguity?